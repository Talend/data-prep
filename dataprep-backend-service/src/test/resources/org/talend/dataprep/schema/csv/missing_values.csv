character;actor;active
Sheriff Woody;Tom Hanks;1995–present

Buzz Lightyear;;1995–present
Mr. Potato Head;Don Rickles;1995–present
Mrs. Potato Head;;
Barbie;Jodi Benson;1999-present