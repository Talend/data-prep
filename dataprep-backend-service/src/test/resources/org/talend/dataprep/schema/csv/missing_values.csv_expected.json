[
  {
    "0000": "Sheriff Woody",
    "0001": "Tom Hanks",
    "0002": "1995–present"
  },
  {
    "0000": "Buzz Lightyear",
    "0001": "",
    "0002": "1995–present"
  },
  {
    "0000": "Mr. Potato Head",
    "0001": "Don Rickles",
    "0002": "1995–present"
  },
  {
    "0000": "Mrs. Potato Head",
    "0001": "",
    "0002": ""
  },
  {
    "0000": "Barbie",
    "0001": "Jodi Benson",
    "0002": "1999-present"
  }
]